sourcegraph/sourcegraph
# JetBrains plugin: allow new-style tokens

Redo of https://github.com/sourcegraph/sourcegraph/pull/52188

Previously, the Sourcegraph plugin didn't allow new-style access tokens
that start with `sgp_` and, therefore, have 44 characters instead of 40
characters. Now, the plugin treats these tokens as valid.

## Test plan

- Green CI.
- Run the plugin and paste in a token like sgp_1111111111111111111111111111111111111111 into the settings
- The plugin should not report an error
- Paste an invalid token like sgp_111111111111111111111111111111111111111
- The plugin should report an error about the token being invalid
